.github/workflows/pr-build.yml

```diff
@@ -28,12 +28,32 @@ jobs:
           restore-keys: |
             gradle-packages-${{ runner.os }}-${{ github.job }}
             gradle-packages-${{ runner.os }}
-      
+
       - name: Build with Gradle
         uses: hypertrace/github-actions/gradle@main
-        with: 
+        with:
           args: build dockerBuildImages
 
+      - name: Determine container tag
+        id: tag
+        run: echo tag=$(./gradlew -q printDockerImageDefaultTag | head -1) >> $GITHUB_OUTPUT
+
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@master
```

Comment:
Same thing here - if we want this to be a standard, we can make a shared action with these 20 or so lines.

Reply:
Added github action - https://github.com/hypertrace/github-actions/pull/9